src/components/YourCompany/YourCompany.vue

```diff
@@ -25,40 +25,43 @@
         </v-flex>
 
         <template v-if="!isEditingNames">
-          <v-flex xs6>
-            <div class="company-name">{{ companyName }}</div>
+          <v-flex xs6 class="mt-n2">
+            <div class="company-name font-weight-bold">{{ companyName }}</div>
             <template v-if="companyNameChanges && !hasNewNr">
               <div class="company-info mt-4">
-                <span class="font-weight-bold">Business Type: </span>
+                <span class="subtitle">Business Type: </span>
                 <span>{{getEntityDesc(getEntityType)}}</span>
               </div>
               <div class="company-info">
                 <span>The name of this business will be the current Incorporation Number followed by "B.C. Ltd."</span>
               </div>
             </template>
             <template v-if="isAlteration() && hasNewNr">
+              <div class="company-name">{{ getNameRequest.nrNumber }}</div>
               <div class="company-info mt-4">
-                <span class="font-weight-bold">Business Type: </span>
-                <span>{{getEntityDesc(getNameRequest.legalType)}}</span>
+                <span class="subtitle">Business Type: </span>
+                <span :class="{ 'hasConflict': isConflictingLegalType}">{{getEntityDesc(getNameRequest.legalType)}}
+                </span>
                 <v-tooltip top content-class="top-tooltip" transition="fade-transition">
                   <template v-slot:activator="{ on }">
-                    <v-icon v-if="isConflictingLegalType" v-on="on" class="ml-2" color="red darken-3" small>
+                    <v-icon v-if="isConflictingLegalType" v-on="on" class="ml-2" color="#d3272c" small>
```

Comment:
Again, can we use a Vuetify theme colour here?

Reply:
100% will do